source/update.rst

```diff
@@ -140,30 +140,39 @@ upgraded, the Operator updates the CR to reflect the new image paths and carries
 on sequential Pods deletion in a safe order, allowing StatefulSet to redeploy
 the cluster Pods with the new image.
 
+.. note:: Being enabled, Smart Update will force the Operator to take MongoDB
+   version from Version Service and not from the ``mongod.image`` option during
+   the very first start of the cluster.
+
 The upgrade details are set in the ``upgradeOptions`` section of the 
 ``deploy/cr.yaml`` configuration file. Make the following edits to configure
 updates:
 
 #. Set the ``apply`` option to one of the following values:
 
-   * ``Recommended`` - automatic upgrades will choose the most recent version
+   * ``Recommended`` - automatic upgrade will choose the most recent version
      of software flagged as Recommended (for clusters created from scratch,
      the Percona Server for MongoDB 4.4 version will be selected instead of the
-     Percona Server for MongoDB 4.2 or 4.0 version
-     regardless of the image path; for already existing clusters, the 4.4
-     vs. 4.2 or 4.0 branch choice will be preserved),
-   * ``Latest`` - automatic upgrades will choose the most recent version of
+     Percona Server for MongoDB 4.2, 4.0, or 3.6 version regardless of the image
+     path; for already existing clusters, the 4.4 vs. 4.2, 4.0, or 3.6 branch
+     choice will be preserved),
+   * ``4.4-recommended``, ``4.2-recommended``, ``4.0-recommended``,
+     ``3.6-recommended`` - same as above, but preserves specific major MongoDB
+     version for newly provisioned clusters (ex. 4.4 will not be automatically
+     used instead of 4.2),
+   * ``Latest`` - automatic upgrade will choose the most recent version of
      the software available (for clusters created from scratch,
      the Percona Server for MongoDB 4.4 version will be selected instead of the
-     Percona Server for MongoDB 4.2 or 4.0 version
-     regardless of the image path; for already existing clusters, the 4.4
-     vs. 4.2 or 4.0 branch choice will be preserved),
-   * *specific version number* - will apply an upgrade if the running Percona
-     Server for MongoDB
-     version doesn't match the explicit version number with no future upgrades
+     Percona Server for MongoDB 4.2, 4.0, or 3.6 version regardless of the image
+     path; for already existing clusters, the 4.4 vs. 4.2, 4.0, or 3.6 branch
+     choice will be preserved),
+   * ``4.4-latest``, ``4.2-latest``, ``4.0-latest``, ``3.6-latest`` - same as
+     above, but preserves specific major MongoDB version for newly provisioned
+     clusters (ex. 4.4 will not be automatically used instead of 4.2),
+   * *version number* - specify the desired version explicitly
      (version numbers are specified as {{{mongodb44recommended}}},
      {{{mongodb42recommended}}}, etc.),
```

Comment:
I'm just curious, if there is a good place where the user can find the versions he can set.